user/permission.py

```diff
@@ -71,4 +71,31 @@ def verify_permission(
         if claim_action == required_action or claim_action == "manage":
             return True
 
-        return False
\ No newline at end of file
+        return False
+    
+    @classmethod
+    def check_permission(cls, action: PERMISSION_TYPE, target: PERMISSION_TARGET):

```

Comment:
Nice, this will be useful for some routes!
This method can just be called `check`. 
Otherwise it seems good. I'm gonna look at the newer version of the auth library for upgrade